src/mono/mono/mini/aot-compiler.c

```diff
@@ -14142,6 +14151,14 @@ aot_assembly (MonoAssembly *ass, guint32 jit_opts, MonoAotOptions *aot_options)
 		acfg->logfile = fopen (acfg->aot_opts.logfile, "a+");
 	}
 
+	if (acfg->aot_opts.compiled_methods_outpath) {
```

Comment:
Wouldn't be better for the caller of the aot compiler passing in the full path ? It has to compute it anyway if it wants to read it afterwards.

Reply:
Fixed